artifactor/plugins/reporter.py

```diff
@@ -285,9 +291,12 @@ def build_li(self, lev):
                 teststring = '<span name="mod_lev" class="label label-primary">T</span>'
                 label = '<span class="label label-{}">{}</span>'.format(
                     bimdict[v['outcomes']['overall']], v['outcomes']['overall'].upper())
-                link = ('<a href="#{}">{} {} {} '
-                        '<span style="color:#888888"><em>[{}]</em></span></a>').format(
-                    v['name'], os.path.split(v['name'])[1], teststring, label, pretty_time)
+                proc_name = process_pytest_path(v['name'])[-1]
+                link = (
+                    '<a href="#{}">{} {} {} <span style="color:#888888"><em>[{}]</em></span></a>'
+                    .format(v['name'], proc_name, teststring, label, pretty_time))
+                # Do we really need the os.path.split (now process_pytest_path) here?
+                # For me it seems the name is always the leaf
```

Comment:
Why the os.path.split was here @psav ? Could it smuggle more than just the leaf location of the node?